src/Support/Collection.php

```diff
@@ -1778,7 +1778,7 @@ public function toJson($options = 0)
      *
      * @return \ArrayIterator
      */
-    public function getIterator()
+    public function getIterator(): Traversable
```

Comment:
Why not `ArrayIterator`?

Reply:
Because I'm not sure if the return type needs an exact match or not

Reply:
If you change it to `ArrayIterator` I'll approve the PR